debian/mariadb-backup.links

```diff
@@ -1 +1,3 @@
-usr/bin/mariabackup usr/bin/mariadb-backup
+usr/bin/mariadb-backup usr/bin/mariabackup
+
```

Comment:
empty line

Reply:
I ran wrap-and-sort -a -v in new commit cf70893